docker/docker_image_src.go

```diff
@@ -436,7 +438,7 @@ func deleteImage(ctx context.Context, sys *types.SystemContext, ref dockerRefere
 		return err
 	}
 	defer get.Body.Close()
-	manifestBody, err := ioutil.ReadAll(get.Body)
+	manifestBody, err := util.ReadAtMost(get.Body, 4*util.MegaByte)
```

Comment:
There really should be a comment somewhere pointing out that docker/distribution has the same limit.

Reply:
… “really should be” is, on reflection, too strong. We can live without it — but if there are two different values, it will cause anyone new to the code to wonder about reasons for the difference.